### PR TITLE
On branch gh-pages-edburns-msft-jea-254-adjust-release-plan https://d…

### PR DESCRIPTION
**Pull request by edburns**
…ev.azure.com/jakarta-ee-azdo/jakarta-ee-azdo/_workitems/edit/254

modified:   jakartaee11/JakartaEE11ReleasePlan.md

### CODE REVIEW

File: jakartaee11/JakartaEE11ReleasePlan.md [hunk 1]
```diff
@@ -227,13 +234,12 @@ We are proposing to deliver Jakarta EE 11 in a set of waves similar to those del
...
 #### Wave 7
 
-- Jakarta Contexts and Dependency Injection Enterprise Edition (CDI EE)
 - Jakarta Security
 - Jakarta Data
 
 #### Wave 8
 
-- Jakarta Platform
+- Jakarta Platform (including content formery in CDI EE)
```
Comment (mswatosh) [inline jakartaee11/JakartaEE11ReleasePlan.md:242]:
```suggestion
- Jakarta Platform (including content formerly in CDI EE)
```

File: jakartaee11/JakartaEE11ReleasePlan.md [hunk 2]
```diff
@@ -4,18 +4,28 @@
...
+| *All*        |              |         |             |         | M2 release | | |
+| *All*        |              |         |             |         | Wave 5 specs release review by 2024-02-29 | | |
+| *All*        |              |         |             |         | M3 release | | |
+| *All*        |              |         |             |         | Wave 6, 7 specs release review by 2024-03-29 | | |
+| *All*        |              |         |             |         | M4 release | | |
+| *All*        |              |         |             |         | TCK pass on Java SE 21 |          | |
+| *Components* |              |         |             |         | Individual Component Spec Ballots | | |
+| *Platform*   |              |         |             |         | Platform TCK pass on Java SE 21 | | |
+| *Components* |              |         |             |         |         | Complete implementations | |
+| *Platform*   |              |         |             |         |         |         | Platform ballot |
```
Comment (Emily-Jiang) [inline jakartaee11/JakartaEE11ReleasePlan.md:23]: Where are Web Profile and Core Profile ballots?

File: jakartaee11/JakartaEE11ReleasePlan.md [hunk 3]
```diff
@@ -4,18 +4,28 @@
...
-| *Platform*   | | | | | Platform TCK pass on Java SE 21 | |
-| *Platform*   | | | | | | Platform ballot |
-| *Platform*   | | | | | | **Release** | |
+|              | Q1 2023      | Q2 2023 | Q3 2023     | Q4 2023 | Q1 2024 | Q2 2024 | June/July 2024  |
+|--------------|--------------|---------|-------------|---------|---------|---------|-----------------|
+| *Components* | Plan Reviews |         |             |         |         |         |                 |
+| *Platform*   |              |         | Plan Review |         |         |         |                 |
+| *All*        |              |         |             | TCK pass w/Security Manager Disabled |    | | |
+| *All*        |              |         |             | M1 release |      |         |                 |
+| *All*        |              |         |             |         | Wave 1, 2, 3 specs release review by 2024-01-30 | | |
```
Comment (mswatosh) [inline jakartaee11/JakartaEE11ReleasePlan.md:13]:
```suggestion
| *All*        |              |         |             |         | Wave 1, 2, 3, 4 specs release review by 2024-01-30 | | |
```

File: jakartaee11/JakartaEE11ReleasePlan.md (hunk 1 above)
Comment (Emily-Jiang) [reply jakartaee11/JakartaEE11ReleasePlan.md:242]: We need another line for Web Profile.

File: jakartaee11/JakartaEE11ReleasePlan.md (hunk 3 above)
Comment (edburns) [reply jakartaee11/JakartaEE11ReleasePlan.md]: Fixed in 790c3148b2da551ec9a02300d4883f3c50caecde.

**Pull request merged**